src/GraphQL/Schemas/Primary/Mutations/MarkAndListPendingTransactionsMutation.php

```diff
@@ -78,9 +78,11 @@ function (Builder $query) use ($args) {
                     return $query->whereIn('wallet_public_key', $publicKeys);
                 },
                 function (Builder $query) {
-                    return $query
-                        ->whereIn('wallet_public_key', Account::managedPublicKeys())
-                        ->orWhereNull('wallet_public_key')->where('state', '=', TransactionState::PENDING->name);
+                    $query->where(
+                        fn ($subquery) => $subquery
+                            ->whereIn('wallet_public_key', Account::managedPublicKeys())
+                            ->orWhereNull('wallet_public_key')
+                    );
```

Comment:
Why have you removed TransactionState::PENDING ?

Reply:
That condition is redundant, this is how the query looks like
Before
```
select `id`, `wallet_public_key`, `id`, `transaction_chain_id`, `transaction_chain_hash`, `state`, `encoded_data` 
from `transactions` 
where 
(`state` = 'PENDING'  and `wallet_public_key` in ('0x3df992cff39a820dfb3c3dff58845599bffc4bc191fb7c898b5f0f777afb9928')  or `wallet_public_key` is null and `state` = 'PENDING') 
and `transactions`.`id` in (select `model_id` from `transactions_tenant` where `user_id` = 1) 
order by `id` asc limit 16
```

After
```
select `id`, `wallet_public_key`, `id`, `transaction_chain_id`, `transaction_chain_hash`, `state`, `encoded_data` 
from `transactions` 
where 
`state` = 'PENDING' 
and (`wallet_public_key` in ('0x3df992cff39a820dfb3c3dff58845599bffc4bc191fb7c898b5f0f777afb9928') or `wallet_public_key` is null) 
and `transactions`.`id` in (select `model_id` from `transactions_tenant` where `user_id` = 1) 
order by `id` asc limit 16
```